common/app/common/Edition.scala

```diff
@@ -80,9 +82,15 @@ object Edition {
     // in production no cookies make it this far
     val editionFromCookie = request.cookies.get("GU_EDITION").map(_.value)
 
+    val participatingInTest =
+      ActiveExperiments.isParticipating(EuropeNetworkFront)(request) || EuropeNetworkFrontSwitch.isSwitchedOn
+
     editionFromParameter
       .orElse(editionFromHeader)
       .orElse(editionFromCookie)
+      // Fastly does not have switch information so we will always try and set the edition to Europe
+      // if a user is in CoE and then fallback to INT edition in Frontend if that user is not part of the experiment.
+      .map(edition => if (edition.equals("EUR") && !participatingInTest) "INT" else edition)
```

Comment:
In theory we're creating 2 new caches with the changes to the VCL.

- A new Europe experiment cache, where all users in the experiment will share 1 cache.
- A new Europe edition cache which is shared by all non-experiment users in the edition.

My initial worry is that by using both a switch and an experiment that we could pollute one of the caches, but I don't think this is an issue.

Reply:
Doesn't look like we're ever going to increase this [percentage](https://github.com/guardian/frontend/blob/main/common/app/experiments/Experiments.scala#L53), right? As I understand the "Europe experiment cache" will be only accessed by people who have already opted in (Guardian employees) and the "Europe edition cache" by all the rest (as you say non-variant experiment users).

Reply:
> Doesn't look like we're ever going to increase this [percentage](https://github.com/guardian/frontend/blob/main/common/app/experiments/Experiments.scala#L53), right?

Thats right!